isobus/src/can_NAME.cpp

```diff
@@ -79,10 +84,12 @@ namespace isobus
 
 	void NAME::set_function_code(std::uint8_t value)
 	{
-		uint64_t tempNAME = rawName;
-		tempNAME &= ~static_cast<std::uint64_t>(0xFF0000000000);
-		tempNAME |= (static_cast<std::uint64_t>(value & 0xFF) << 40);
-		set_full_name(tempNAME);
+		if (value > 0xFF)
```

Comment:
Ah I didn't catch this before I merged it, but this check does nothing, as a uint8_t cannot be > 0xFF, so some compilers will throw a warning

Reply:
Oh I see, I can remove it in the other PR if you want?

Reply:
Yeah if you don't mind that'd be good